Lib/test/test_typing.py

```diff
@@ -560,8 +560,9 @@ def test_no_multiple_subscripts(self):
             Literal[1][1]
 
     def test_equal(self):
+        self.assertEqual(Literal[1], Literal[1])
         self.assertEqual(Literal[1, 2], Literal[2, 1])
-        self.assertNotEqual(Literal[1, True], Literal[1])
```

Comment:
I think we should keep the deleted test, it may be useful so that there isn't an accidental regression by someone else in the future.

Reply:
Thanks, I will return this test